deploy_templates.py

```diff
@@ -65,7 +66,7 @@
   "hdfs_data_dirs": os.getenv("HDFS_DATA_DIRS"),
   "mapred_local_dirs": os.getenv("MAPRED_LOCAL_DIRS"),
   "spark_local_dirs": os.getenv("SPARK_LOCAL_DIRS"),
-  "default_spark_mem": "%dm" % spark_mb,
+  "default_spark_mem": "%dm" % slave_ram_mb,
```

Comment:
can we rename this variable to spark_executor_mem ? I think that is more clear. Also you'll need to update `spark-defaults.conf` with this new variable